contracts/PetPark.sol

```diff
@@ -4,4 +4,101 @@ pragma solidity ^0.8.0;
 
 contract PetPark {
 
-}
\ No newline at end of file
+enum animalType {None, Fish, Cat, Dog, Rabbit, Parrot}
+enum gender {Male, Female}
+address immutable owner;
+
+struct member{
+    gender _gender;
+    uint age;
+    address memberAddress;
+}
+mapping (animalType => uint) numberOfAnimals;
+mapping (address => animalType) borrowed;
+mapping (address => member) members;
+
+event Added (animalType, uint);
+event Borrowed (animalType);
+event Returned (animalType);
+
+constructor (){
+  owner = msg.sender;
+}
+
+function add (animalType _animalType, uint _count) public{
+    require (owner == msg.sender, "Not an owner");
```

Comment:
Better to use a modifier here for checking owner as it provides better code readability